sdk/tables/azure-data-tables/README.md

```diff
@@ -139,18 +119,18 @@ use of a dedicated client object.
 ### Clients
 Two different clients are provided to interact with the various components of the Table Service:
 1. **`TableServiceClient`** -
-    this client represents interaction with the Azure account itself, and allows you to acquire preconfigured
-    client instances to access the tables within. It provides operations to retrieve and configure the account
-    properties as well as query, create, and delete tables within the account. To perform operations on a specific table,
-    retrieve a client using the `get_table_client` method.
+    * Interacts with the Azure account itself
+    * Allows you to acquire preconfigured client instances to access the tables within.
+    * Provides operations to retrieve and configure the account
+    * Query, create, and delete tables within the account.
+    * To perform operations on a specific table, retrieve a client using the `get_table_client` method.
```

Comment:
I like this list - though maybe we could simplify it even further to just the core concepts:
```
    * Get and set account settings.
    * Query, create, and delete tables within the account.
    * Access a specific table using the `get_table_client` method.
```